docs/cisco.ios.ios_user_module.rst

```diff
@@ -548,38 +685,139 @@ Examples
         privilege: 15
         state: present
 
-    - name: Set user view/role
-      cisco.ios.ios_user:
-        name: netop
-        view: network-operator
-        state: present
+    # Commands Fired:
+    # ---------------
+
+    # "commands": [
+    #     "username netop privilege 15",
+    #     "username netend privilege 15"
+    # ],
+
+    # After state:
+    # ------------
+
+    # router-ios#show running-config | section username
+    # username admin privilege 15 password 0 password
+    # username netop privilege 15 password 0 password1
+    # username netend privilege 15 password 0 password2
+
+    # Using state: present
+
+    # Before state:
+    # -------------
+
+    # router-ios#show running-config | section ^username
+    # username admin privilege 15 password 0 password
+    # username netop privilege 15 password 0 oldpassword
+
+    # Present state Change Password for User netop play:
+    # --------------------------------------------
 
     - name: Change Password for User netop
       cisco.ios.ios_user:
         name: netop
-        configured_password: "{{ new_password }}"
+        configured_password: "newpassword"
+        password_type: password
         update_password: always
         state: present
 
-    - name: Aggregate of users
+    # Commands Fired:
+    # ---------------
+
+    # "commands": [
+    #     "username netop password newpassword"
+    # ],
+
+    # After state:
+    # ------------
+
+    # router-ios#show running-config | section username
+    # username admin privilege 15 password 0 password
+    # username netop privilege 15 password 0 newpassword
+
+    # Using state: present
+
+    # Before state:
+    # -------------
+
+    # router-ios#show running-config | section ^username
+    # username admin privilege 15 password 0 password
+    # username netop privilege 15 password 0 password
+    # username netend privilege 15 password 0 password
+
+    # Present state set user view/role for users play:
+    # --------------------------------------------
+
+    - name: Set user view/role for users
       cisco.ios.ios_user:
         aggregate:
-          - name: ansibletest2
-          - name: ansibletest3
+          - name: netop
+          - name: netend
         view: network-admin
+        state: present
 
-    - name: Add a user specifying password type
-      cisco.ios.ios_user:
-        name: ansibletest4
-        configured_password: "{{ new_password }}"
-        password_type: password
+    # Commands Fired:
```

Comment:
This should be Task Output, make it in sync with other collection doc
look at interface modules for reference,